schemas/v1/create-hook-request.yml

```diff
@@ -4,6 +4,27 @@ description: |
   Definition of a hook that can create tasks at defined times.
 type:                       object
 properties:
+  bindings:
+    title:                'Bindings List'
+    description: |
+      List of bindings for the hook with given `hookGroupId` and `hookId`.
+    type:                 array
+    items:
+      title:              "Binding"
+      description: |
+        Exchange and RoutingKeyPattern for each binding
+      type:               object
+      properties:
+        exchange:
+          type:           string
+        routingKeyPattern:
+          type:           string
+        routingKeyReference:
+          type:           string
```

Comment:
Let's leave this off -- it's used by pulse consumers to disassemble the routing key into a data structure, but we won't be doing that here.